src/backend/metal/src/conversions.rs

```diff
@@ -132,6 +153,661 @@ impl PrivateCapabilities {
+                optimal_tiling: If::SAMPLED_LINEAR,
+                ..defaults
+            },
+            BC4_RSnorm if self.format_bc => Properties {
+                optimal_tiling: If::SAMPLED_LINEAR,
+                ..defaults
+            },
+            BC5_RGUnorm if self.format_bc => Properties {
+                optimal_tiling: If::SAMPLED_LINEAR,
+                ..defaults
+            },
+            BC5_RGSnorm if self.format_bc => Properties {
+                optimal_tiling: If::SAMPLED_LINEAR,
+                ..defaults
+            },
+            BC6H_RGBUfloat if self.format_bc => Properties {
+                optimal_tiling: If::SAMPLED_LINEAR,
+                ..defaults
+            },
+            BC6H_RGBFloat if self.format_bc => Properties {
+                optimal_tiling: If::SAMPLED_LINEAR,
+                ..defaults
+            },
+            BC7_RGBAUnorm if self.format_bc => Properties {
+                optimal_tiling: If::SAMPLED_LINEAR,
+                ..defaults
+            },
+            BC7_RGBAUnorm_sRGB if self.format_bc => Properties {
+                optimal_tiling: If::SAMPLED_LINEAR,
+                ..defaults
+            },
+            Depth16Unorm if self.format_depth16unorm => Properties {
+                optimal_tiling: If::SAMPLED_LINEAR,
+                ..defaults
+            },
+            Depth32Float if self.format_depth32float_filter => Properties {
+                optimal_tiling: If::SAMPLED_LINEAR,
+                ..defaults
+            },
+            Depth32Float if self.format_depth32float_none => Properties {
+                optimal_tiling: If::empty(),
+                ..defaults
+            },
+            Stencil8 => Properties {
+                optimal_tiling: If::empty(),
+                ..defaults
+            },
+            Depth24Unorm_Stencil8 if self.format_depth24_stencil8 => Properties {
+                optimal_tiling: If::SAMPLED_LINEAR,
+                ..defaults
+            },
+            Depth32Float_Stencil8 if self.format_depth32_stencil8_filter => Properties {
+                optimal_tiling: If::SAMPLED_LINEAR,
+                ..defaults
+            },
+            Depth32Float_Stencil8 if self.format_depth32_stencil8_none => Properties {
+                optimal_tiling: If::empty(),
+                ..defaults
+            },
+            BGR10A2Unorm if self.format_bgr10a2_all => Properties {
+                optimal_tiling: If::SAMPLED_LINEAR
+                    | If::STORAGE
+                    | If::COLOR_ATTACHMENT
+                    | If::COLOR_ATTACHMENT_BLEND,
+                ..defaults
+            },
+            BGR10A2Unorm if self.format_bgr10a2_no_write => Properties {
+                optimal_tiling: If::SAMPLED_LINEAR
+                    | If::COLOR_ATTACHMENT
+                    | If::COLOR_ATTACHMENT_BLEND,
+                ..defaults
+            },
+            _ => defaults,
```

Comment:
are those the compressed formats being excluded?

Reply:
I believe so.